package.json

```diff
@@ -80,8 +80,8 @@
     "react-native-webview": "^7.5.2",
     "react-navigation": "^4.0.10",
     "react-navigation-drawer": "^2.3.3",
-    "react-navigation-redux-helpers": "^2.0.8",
-    "react-navigation-stack": "1.10.3",
+    "react-navigation-redux-helpers": "^4.0.1",
```

Comment:
According to [this](https://github.com/react-navigation/redux-helpers/releases/tag/3.0.0) or later changelogs, don't we need `yarn add @react-navigation/core` ?

Reply:
i am not sure, i didnt see anything about that on readme. and it is working as expected right now

Reply:
ok